lib/erb-helper.coffee

```diff
@@ -23,6 +24,9 @@ module.exports =
   comment: ->
     @insertTag('<%#  %>')
 
+  end: ->
+    @insertTag('<% end %>')
+
   insertTag: (tag) ->
```

Comment:
Add a second parameter here, maybe called `isBlockTag` and default it to false.

Modify the if in this function to first check for `isBlockTag`, if so then just call `editor.insertText(tag)`